backend/flask_api.py

```diff
@@ -128,16 +130,25 @@ def get_project_by_id(project_id):
 
 @app.route('/api/projects/<uuid:project_id>', methods=['DELETE'])
 def delete_project(project_id):
-    if coll.delete_one({'_id':project_id}).deleted_count != 0:
-        return make_response('Success')
+    try:
+        es.delete(index="projects-index", doc_type='Project', id=project_id, refresh=True)
+    except NotFoundError:
+        if coll.delete_one({'_id':project_id}).deleted_count == 0:
+            return make_response('Project not found', 404)
+        else:
+            return make_response('Success')
     else:
-        return make_response('Project not found', 404)
+        coll.delete_one({'_id':project_id})
+        return make_response('Success')
```

Comment:
I feel like we should return a summary where the delete was successful and where not. Returning "success" if we had database inconsitency doesn't seem right.